util/testutil/integration/dockerd.go

```diff
@@ -1,45 +1,66 @@
 package integration
 
 import (
+	"bytes"
 	"context"
+	"fmt"
 	"io"
 	"net"
 	"os"
-	"os/exec"
 	"path/filepath"
-	"strings"
-	"syscall"
-	"testing"
 	"time"
 
 	"github.com/docker/docker/client"
-	"github.com/moby/buildkit/identity"
+	"github.com/moby/buildkit/util/testutil/dockerd"
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 	"golang.org/x/sync/errgroup"
 )
 
-const (
-	shortLen      = 12
-	dockerdBinary = "dockerd"
-)
-
 // InitDockerdWorker registers a dockerd worker with the global registry.
 func InitDockerdWorker() {
-	Register(&dockerd{})
+	Register(&moby{
+		name:     "dockerd",
+		rootless: false,
+		features: newFeatures(
+			FeatureCacheExport,
+			FeatureCacheImport,
+			FeatureDirectPush,
+			FeatureImageExporter,
+			FeatureMultiCacheExport,
+			FeatureMultiPlatform,
+			FeatureOCIExporter,
+			FeatureOCILayout,
+			FeatureProvenance,
+			FeatureSBOM,
+			FeatureSecurityMode,
+		),
+	})
+	Register(&moby{
+		name:     "dockerd-containerd",
+		rootless: false,
+		features: newFeatures(
+			FeatureProvenance,
```

Comment:
Why is provenance & SBOM here. I think it is a out-of-sync error again. Tests need to be compatible with the same commit tests are part of not with some older version part of some vendor.